src/components/List.js

```diff
@@ -178,9 +178,13 @@ const List = ({ firestore }) => {
       />
       {showModal && (
         <Modal setDisplay={setModalDisplay}>
-          <h1>Warning</h1>
           <p>Are you sure you want to delete this item?</p>
-          <button onClick={() => deleteItem()}>Yes</button>
+          <button
+            className="text-primary hover:underline"
```

Comment:
Is `hover:underline` a valid class name? 🤔

Reply:
@segdeha Yes, [tailwind](https://tailwindcss.com/docs/text-decoration/#hover) provides us with it.